details/CLUSTER_DEFINITION.md

```diff
@@ -136,6 +150,7 @@ After creation using the definition above, the code below shows the completed cl
 - `workers`: Array of worker definition objects. Each array item represents one worker node. In order to create a cluster with three worker nodes, this array MUST have three items, even if all worker share the same configuration.
 - `workers[n].aws.instance_type`: Name of the EC2 instance type to use for the worker node. For clusters running on AWS, this attribute is required on cluster creation and must have the same value for all worker nodes of the cluster.
 - `workers[n].memory`: Memory definition object. This object can currently contain only one attribute `size_gb`, which indicates the RAM size in Gigabytes as an integer. For clusters running on AWS, this attribute is ignored on cluster creation.
-- `workers[n].storage`: Storage definition object. Storage here refers to the local disk storage of a worker node. The definition object can currently contain only one attribute `size_gb`, which indicates the local storage size in Gigabytes as an integer. For clusters running on AWS, this attribute is ignored on cluster creation.
+- `workers[n].volumes`: Volumes definition object. The volumes are ephemeral storage that is attached to each worker node. 
+- `workers[n].volumes[n].docker`: Docker volume size definition object. This refers to the size of the disk attached to the node where docker images are stored. This volume is usually attached to `/var/lib/docker` and the docker daemon uses this location for storing images. The definition object can currently contain only one attribute `size_gb`, which indicates the docker volume size in Gigabytes as an integer. 
```

Comment:
`volumes[n].docker` indicates that `volumes` is an array, which it isn't. Please change to `volumes.docker`.

Reply:
> ... for storing images.

See above comment regarding usage of this volume

Reply:
resolved